src/adv.py

```diff
@@ -38,14 +38,38 @@
 #
 
 # Make a new player object that is currently in the 'outside' room.
-
+player = Player('Irving', room['outside'])
+print(player)
+# userInput = ""
 # Write a loop that:
 #
+while True:
+
 # * Prints the current room name
+    print(player.location.name)
 # * Prints the current description (the textwrap module might be useful here).
+    print(player.location.description)
 # * Waits for user input and decides what to do.
-#
+    user_input = input(
+        "where do you want go? (N)orth, (S)outh, (E)ast, or (W)est?")
 # If the user enters a cardinal direction, attempt to move to the room there.
+    if(user_input == 'n' or user_input == 'N'):
+        target = player.location.n_to
```

Comment:
what happens if there is nothing 'n_to' from the players location? you'll want to ad some code to check for and handle this case